include/mbedtls/ccm.h

```diff
@@ -38,68 +48,71 @@ extern "C" {
 #endif
 
 /**
- * \brief          CCM context structure
+ * \brief   The CCM context-type definition. The CCM context is passed 
+ *			to the APIs called.
```

Comment:
I would prefer to omit the second sentence, or, if not, to make the indentation uniform.

Reply:
fixed indentation. I know it seems obvious to us, but some customers need basic concepts.